src/lib/components/editors/code-editor/terra-code-editor.component.ts

```diff
@@ -38,7 +38,7 @@ export class TerraCodeEditorComponent extends TerraBaseEditorComponent implement
     @Language()
     public _lang: string;
 
-    public _viewConfirmation: { primaryButton: TerraButtonInterface; secondaryButton: TerraButtonInterface; };
+    public _viewConfirmation: { primaryButton: TerraButtonInterface; secondaryButton: TerraButtonInterface };
```

Comment:
die Änderung ergibt für mich keinen Sinn

Reply:
Scheinbar ist das Semikolon am Ende eines Inlineinterfaces nicht nötig. Ähnlich wie ein Komma am Ende eines Objektes nicht nötig ist. Daher, weil unnötig, wurde es entfernt. 

Andererseits wollten wir eigentlich auch keine Inlineinterfaces haben 🤷‍♂️ Wäre vermutlich gut das in einem eigenen PR zu refactorn :)

Reply:
ja, das ist akzeptabel. können ja einfach zwei Variablen werden. also können wir in dem PR erstmal so lassen